src/main/java/website/frontrow/Launcher.java

```diff
@@ -62,7 +62,7 @@ public static void main(String[] args) throws IOException
      * @param filename The file name of the level to load.
      * @param playerCount The amount of players in this game.
      */
-    @SuppressWarnings("methodlength") // We need to make a GameFactory and UIBuilder
+    @SuppressWarnings("methodlength")
```

Comment:
We already fixed this problem in the key bindings branch. 

Verstuurd vanaf mijn iPhone

> Op 30 okt. 2015 om 17:30 heeft ThatFlint <notifications@github.com> het volgende geschreven:
> 
> In src/main/java/website/frontrow/Launcher.java:
> 
> > @@ -62,7 +62,7 @@ public static void main(String[] args) throws IOException
> >       * @param filename The file name of the level to load.
> >       * @param playerCount The amount of players in this game.
> >       */
> > -    @SuppressWarnings("methodlength") // We need to make a GameFactory and UIBuilder
> > +    @SuppressWarnings("methodlength")
> I made it shorter by creating a new method, keeping it organized and easy to read.
> 
> —
> Reply to this email directly or view it on GitHub.
>